src/test/java/no/hvl/dat250/h2020/group5/repostories/PollRepositoryTest.java

```diff
@@ -83,17 +82,25 @@ public void shouldDeleteVotesWhenDeletingPollTest() {
   public void shouldOnlyDeleteVotesLinkedToPollWhenDeletingPollTest() {
     Vote voteNotLinkedToPoll = new Vote();
     voteRepository.save(voteNotLinkedToPoll);
-    user.detachPoll(poll);
-    pollRepository.delete(poll);
+    savedUser.detachPoll(savedPoll);
+    pollRepository.delete(savedPoll);
     Assertions.assertEquals(1, voteRepository.count());
     Assertions.assertEquals(voteNotLinkedToPoll.getId(), voteRepository.findAll().get(0).getId());
   }
 
   @Test
   public void shouldNotDeleteUserWhenDeletingPollTest() {
-    user.detachPoll(poll);
-    pollRepository.delete(poll);
-    Assertions.assertEquals(user.getId(), userRepository.findAll().get(0).getId());
+    savedUser.detachPoll(savedPoll);
+    pollRepository.delete(savedPoll);
+    Assertions.assertEquals(savedUser.getId(), userRepository.findAll().get(0).getId());
+  }
```

Comment:
Flott at du tester dette. Da bør vi også legge til `owner.detachPoll(poll)` i `pollService.deletePoll()`, for hvis man fjerner `savedUser.detachPoll(savedPoll)` her i testen så feiler den. :)